netci.groovy

```diff
@@ -86,6 +86,11 @@ class Constants {
                'tailcallstress'                 : ['COMPlus_TailcallStress' : '1'],
                'jitsse2only'                    : ['COMPlus_EnableAVX' : '0', 'COMPlus_EnableSSE3_4' : '0'],
                'jitnosimd'                      : ['COMPlus_FeatureSIMD' : '0'],
+               'jitnox86hwintrinsic'            : ['COMPlus_EnableIncompleteISAClass' : '1', 'COMPlus_EnableSSE' : '0' , 'COMPlus_EnableSSE2' : '0' , 'COMPlus_EnableSSE3' : '0' , 'COMPlus_EnableSSSE3' : '0' , 'COMPlus_EnableSSE41' : '0' , 'COMPlus_EnableSSE42' : '0' , 'COMPlus_EnableAVX' : '0' , 'COMPlus_EnableAVX2' : '0' , 'COMPlus_EnableAES' : '0' , 'COMPlus_EnableBMI1' : '0' , 'COMPlus_EnableBMI2' : '0' , 'COMPlus_EnableFMA' : '0' , 'COMPlus_EnableLZCNT' : '0' , 'COMPlus_ EnablePCLMULQDQ' : '0' , 'COMPlus_EnablePOPCNT' : '0'],
+               'jitx86hwintrinsic'              : ['COMPlus_EnableIncompleteISAClass' : '1'],
+               'jitx86hwintrinsicsseencoding'   : ['COMPlus_EnableIncompleteISAClass' : '1', 'COMPlus_EnableAVX' : '0'],
+               'jitx86hwintrinsicnoavx2'        : ['COMPlus_EnableIncompleteISAClass' : '1', 'COMPlus_EnableAVX2' : '0'],
+               'jitx86hwintrinsicnosimd'        : ['COMPlus_EnableIncompleteISAClass' : '1', 'COMPlus_FeatureSIMD' : '0'],
```

Comment:
I think this job will fail until https://github.com/dotnet/coreclr/issues/15641 is resolved

Reply:
Nevermind, this is controlling the JIT Flag, rather than the compiler defines.

In either case, I think the consensus was to decouple these in the long run, so I'm not sure we should add this job in particular

Reply:
No, the environment variable is different from the compile-flag. `FEATURE_SIMD`

Reply:
> so I'm not sure we should add this job in particular

Now, this job tests `Vector128/256<T>` without the underlying SIMD register support (just struct).

Reply:
is that a scenario we need/want to support?

Reply:
I encountered failures from this issue... 
For example, `gtNewMustThrowException` has to consider `Struct` and `SIMD` both https://github.com/dotnet/coreclr/blob/master/src/jit/gentree.cpp#L17910

Reply:
> For example, gtNewMustThrowException has to consider Struct and SIMD both 

This scenario can be covered by "jitnox86hwintrinsic". 
But we have the CI job "nosimd", I just wnted to match it.

Reply:
I think that scenario will end up falling out once `TYP_SIMD` is decoupled from `FEATURE_SIMD` (as per #15641).

That being said, if there are failure cases until it is decoupled, having this job is probably good.

Could you add a comment indicating as such?

Reply:
> Could you add a comment indicating as such?

Will do.